### PR TITLE
Create notifications-Examples.md

### PR DESCRIPTION
**Pull request by Men-hau**
new page

### CODE REVIEW

File: docs/getting_started/notifications-Examples.md [hunk 1]
```diff
@@ -0,0 +1,182 @@
+## Examples
+
+When your conditions are configured, you'll have to choose which action should be triggered by the notification mechanism. You can enable one or more action for each notification rule.
```
Comment (ka0ula) [inline docs/getting_started/notifications-Examples.md:3]:
```suggestion
When your conditions are configured, you'll have to choose which actions should be triggered by the notification mechanism. You can enable one or more actions for each notification rule.
```

File: docs/getting_started/notifications-Examples.md [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+
+For example you can decide to send an e-mail and to send a message on a Mattermost channel.
```
Comment (ka0ula) [inline docs/getting_started/notifications-Examples.md:5]:
```suggestion
For instance, you can decide to send an e-mail AND send a message on a Mattermost channel.
```

File: docs/getting_started/notifications-Examples.md [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+
+Mattermost is a popular professional chat service. The “Mattermost notification” will send messages to any Mattermost instance.  To do so, you’ll have to configure a new `Incoming Webhook` on your Mattermost instance and choose to which Mattermost channel the message should be sent to.
+
+You can refer to the Mattermost documentation on how to [create a new Mattermost “incoming webhook”](https://docs.mattermost.com/developer/webhooks-incoming.html).
+
+### WebHook Notification
+
+The “WebHook notification” will let you send message to interact with third party softwares. Technical informations related to the event will be pushed to an HTTP server. That latter will be able to understand the received payload and act (for example, retrieve more information about the event itself via SEKOIA.IO APIs and then push notification to an internal messaging service).
+
+!!! info You can’t use the WebHook notification mechanism to push information directly to third parties (such as Slack or Telegram), you have to use an intermediate server. To do so, you can use solutions like IFTTT or a simple HTTP server (see below).
```
Comment (ka0ula) [inline docs/getting_started/notifications-Examples.md:28]:
```suggestion
!!! info 
    You can’t use the WebHook notification mechanism to push information directly to third parties (such as Slack or Telegram), you have to use an intermediate server. To do so, you can use solutions like IFTTT or a simple HTTP server (see below).
```

File: docs/getting_started/notifications-Examples.md [hunk 4]
```diff
@@ -0,0 +1,182 @@
...
+
+
+@app.post("/hooks/67a7472b-b43c-415d-b1ce-c135fb1dfb0e") # (1)
+async def handle_new_alert(
+    event: SEKOIAIOWebHookEvent, background_tasks: BackgroundTasks
+):
+    background_tasks.add_task(push_notification_to_teams, event)
+```
+
+1. Entrypoint of our HTTP server that receives SEKOIA.IO WebHook notifications. Please define a unique URL, using and UUID for example.
```
Comment (ka0ula) [inline docs/getting_started/notifications-Examples.md:152]: @Men-hau Is this a summary of what the code is doing?

File: docs/getting_started/notifications-Examples.md [hunk 5]
```diff
@@ -0,0 +1,182 @@
...
+
+export APIKEY="eyJ0eXAiOiJ…" # (3)
+export TEAMS_WEBHOOK_URL="https://…"
+
+poetry run uvicorn main:app # (4)
+
+ngrok http 8000 # (5)
+```
+
+1. Create a Python virtual environment to install our code
```
Comment (ka0ula) [inline docs/getting_started/notifications-Examples.md:176]: @Men-hau same question here, is this a summary?

**Review by ka0ula — CHANGES_REQUESTED** [review]

**Comment by ka0ula** [comment]
This page needs an entry in the mkdocs

File: docs/getting_started/notifications-Examples.md (hunk 4 above)
Comment (Men-hau) [reply docs/getting_started/notifications-Examples.md:153]: these are the steps describe in the original page to make the actions before using the script

File: docs/getting_started/notifications-Examples.md (hunk 5 above)
Comment (Men-hau) [reply docs/getting_started/notifications-Examples.md:177]: these are the steps describe in the original page to make the actions before using the script

**Review by ka0ula — APPROVED** [review]

**Pull request merged**